integration-tests/pytests/test_client_cli.py

```diff
@@ -0,0 +1,135 @@
+import json
+import pytest
+import os
+import time
+from client_cli import Wallet, Transaction
+
+PASSPHRASE = "123456"
+
+def set_env():
+    os.environ['CRYPTO_CHAIN_ID'] = 'test-chain-y3m1e6-AB'
+    current_dir = os.path.dirname(os.path.abspath(__file__))
+    father_dir = os.path.dirname(current_dir)
+    storage_path = os.path.join(father_dir, "data", "wallet")
+    os.environ['CRYPTO_CLIENT_STORAGE'] = storage_path
+    os.environ['CRYPTO_GENESIS_HASH'] = '09A49B6D51D0204198609D632444A54DE261B54C2FC1D4A79B7C3705FBEEB2E3'
```

Comment:
This can be read from the `info.json` like this: https://github.com/crypto-com/chain/blob/master/integration-tests/run.sh#L54